app/src/main/java/org/mozilla/fenix/tabtray/TabTrayController.kt

```diff
@@ -205,6 +205,7 @@ class DefaultTabTrayController(
             newTab = true,
             from = BrowserDirection.FromTabTray
         )
+        metrics.track(Event.SyncedTabOpened)
```

Comment:
The implementation of the Synced Tabs here is behind a secret settings flag and is also going away, so the metrics here would be skewed.

Maybe we wanted to add it to the three-dot menu?